### PR TITLE
Set up React router

### PR DESCRIPTION
**Pull request by malinatran**

### CODE REVIEW

**Comment by sepowitz** [comment]
This looks great. My only concern is that the M O T I needs to persist across pages, so it may be best to just include it right in index.html file that will render all the react components.

**Comment by hhff** [comment]
Agree with @sepowitz 

Feel like each letter should be a component, as they need to spread to all 4 corners.

Also - we've got Babel installed right?  We should be using es6 syntax where possible - I can see a bunch of places - will comment

File: app/config/routes.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var React = require('react');
+var ReactRouter = require('react-router');
+var Router = ReactRouter.Router;
+var Route = ReactRouter.Route;
+var hashHistory = ReactRouter.hashHistory;
+var IndexRoute = ReactRouter.IndexRoute;
+var Main = require('../components/Main');
+var Home = require('../components/Home');
```
Comment (hhff) [inline app/config/routes.js:8]: I'd write this block w/ es6 as:

```
import React from 'react';
import Main from '../components/Main';
import Home from '../components/Home';

import ReactRouter, {
  hashHistory,
  Router,
  Route,
  IndexRoute
}, from 'react-router';
```

File: app/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  render: function() {
+    return (
+      <div>
+        I am home.
+      </div>
+    )
+  }
+});
+
+module.exports = Home;
```
Comment (hhff) [inline app/components/Home.js:13]: es6 style:

```
import { Component } from 'react';

export default class Home extends Component {
  render() {
    return (
      <div>
        I am home.
      </div>
    )
  }
}
```

**Pull request merged**